gammapy/modeling/models/spatial.py

```diff
@@ -371,6 +371,73 @@ def to_region(self, **kwargs):
         )
 
 
+class GeneralizedGaussianSpatialModel(SpatialModel):
+    r"""Two-dimensional Generealized Gaussian model.
+
+    For more information see :ref:`generalized-gaussian-spatial-model`.
+
+    Parameters
+    ----------
+    lon_0, lat_0 : `~astropy.coordinates.Angle`
+        Center position
+    r_eff : `~astropy.coordinates.Angle`
+        Effective radius (semi-major axis if elliptical), in angular units.
+    eta : `float`
+        Shape parameter whitin (0, 1]. Special cases for disk: ->0, Gaussian: 0.5, Laplacian:1
+    e : `float`
+        Eccentricity (:math:`0< e< 1`).
+    phi : `~astropy.coordinates.Angle`
+        Rotation angle :math:`\phi`: of the major semiaxis.
+        Increases counter-clockwise from the North direction.
+    frame : {"icrs", "galactic"}
+        Center position coordinate frame
+    """
+
+    tag = ["GeneralizedGaussianSpatialModel", "gen-gauss"]
+    lon_0 = Parameter("lon_0", "0 deg")
+    lat_0 = Parameter("lat_0", "0 deg", min=-90, max=90)
+    r_eff = Parameter("r_eff", "1 deg")
+    eta = Parameter("eta", 0.5, min=0.01, max=1.0)
+    e = Parameter("e", 0.0, min=0.0, max=1.0, frozen=True)
+    phi = Parameter("phi", "0 deg", frozen=True)
+
+    @staticmethod
+    def evaluate(lon, lat, lon_0, lat_0, r_eff, eta, e, phi):
+        lon = Longitude(lon).wrap_at(f"{np.min(lon)+180*u.deg}")
```

Comment:
Can this be unified / simplified using the `compute_sigma_eff` function we use in the other elliptical models?